src/stylesheets/global.css

```diff
@@ -1,13 +1,70 @@
 /* Global settings and variables */
 
 @import '~normalize.css';
-@import url('https://fonts.googleapis.com/css?family=Lato:300i,300,400,700');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
```

Comment:
The 800 lato font is available at https://fonts.adobe.com/fonts/lato

Reply:
I don't think we can easily use the fonts from Adobe without a subscription to TypeKit. I think it's fine to use the 700 weight in place of the 800 and just let @derekfidler know.